packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -220,6 +232,63 @@ const Breakouts = WebexPlugin.extend({
 
     this.breakouts.set(Object.values(breakouts));
   },
+
+  /**
+   * Make enable breakout resource
+   * @returns {Promise}
+   */
+  checkLocusDTO() {
+    if (this.breakoutServiceUrl) {
+      // @ts-ignore
+      return this.webex
+        .request({
+          method: HTTP_VERBS.POST,
+          uri: this.breakoutServiceUrl,
+          body: {
+            locusUrl: this.locusUrl,
+          },
+        })
+        .catch((err) => {
+          LoggerProxy.logger.error(
+            `Meeting:request#touchBreakout --> Error provisioning error ${err}`
+          );
+          throw err;
+        });
+    }
+
+    return Promise.reject(new Error(`CheckLocusDTO: the breakoutServiceUrl is empty`));
+  },
+
+  /**
+   * Make the meeting enbale or disable breakout session
+   * @param {boolean} enable
+   * @returns {Promise}
+   */
+  async toggleBreakout(enable) {
+    if (this.enableBreakoutSession === undefined) {
+      await this.checkLocusDTO().then(() => {
+        this.doToggleBreakout(enable);
```

Comment:
when I read https://sqbu-github.cisco.com/pages/WebExSquared/breakout/api/resource/enableBreakouts.html my understanding is that the breakout session is enabled after the POST, so no need to do an extra PUT if we're trying to enable it.